packages/@o3r/schematics/src/rule-factories/ng-add/index.ts

```diff
@@ -84,11 +90,16 @@ export function ngAddPackages(packages: string[], options?: NgAddPackageOptions,
     });
     tree.overwrite(packageJsonPath, JSON.stringify(packageJson, null, 2));
 
+    if (options?.skipNgAddSchematicRun) {
+      context.logger.info(`Packages '${packagesToInstall.join(', ')}' were installed. The run of 'ng-add' schematics for these packages is intentionally skipped. You can run them standalone, later.`);
+      return () => tree;
```

Comment:
why not just `return tree;` ?

Reply:
the return type of the parent function forces a Rule